webcompat/views.py

```diff
@@ -147,6 +147,7 @@ def index():
 
 
 @app.route('/issues')
+@cache_policy(private=True, uri_max_age=0, must_revalidate=True)
```

Comment:
Tell me if what I explained here, answer your question.
https://github.com/webcompat/webcompat.com/issues/609#issuecomment-284322939

We specifically want a revalidation at each request, because if we do not do it, we will get the issue with the cached resource (on login/logout state). So no. max-age must always be 0 for revalidation.